src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -228,6 +228,7 @@ public abstract class DLNAResource extends HTTPResource implements Cloneable, Ru
 	private boolean isExternalSubtitlesParsed;
 
 	private double lastTimeSeek = -1.0;
+	protected static final String RECENTLY_ADDED = "Recently Added";
```

Comment:
This will need to be the translated term so like `Messages.getString("RecentlyAdded")`. Will also need the `RecentlyPlayed`, `InProgress` and `MostPlayed` ones